processes/burntmapping_chunks.json

```diff
@@ -0,0 +1,196 @@
+{
+  "namespace": "user",
+  "description": "Classical Normalized Burnt Ratio(NBR) difference is performed using VITO backend on a chunk polygon.",
+  "id": "NBR_chunk_polygons",
```

Comment:
process ID should be the same as the filename

Reply:
updated